rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/PluginDescriptor.java

```diff
@@ -101,6 +105,10 @@ public boolean isFile() {
             return file;
         }
 
+        public boolean isDynamic() {
+            return dynamic;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)